eksctl/bican.jsonnet

```diff
@@ -37,6 +37,15 @@ local notebookNodes = [
       // GPUs in a single AZ are in use and no new nodes can be spawned
       availabilityZones: masterAzs,
     },
+    {
```

Comment:
I _think_ we're missing a  `g4dn.xlarge` here as well? I see it being specified in the profile options, so we should remove it from there if it's not required

Reply:
![image](https://github.com/2i2c-org/infrastructure/assets/3837114/ef9ab233-f149-417a-8157-ea207b7b395d)

It is specified since before, now we specify both here.

I think its reasonable that we transition to a default of specifying both and allowing for a choice of 4CPU and 8CPU, the cost difference is 50%+ for 8 CPU, and when whats matters is just basic testing it could be nice to opt cheap by default or similar.

Reply:
Yeah, Sarah already set that up! https://github.com/2i2c-org/infrastructure/pull/3891/files#diff-0067a696fa6484b21a3052fb489c0e6e09703df1b5c700f5a379d6641e8ff73cR32